scripts/build_go_slim.sh

```diff
@@ -83,7 +83,12 @@ dest_dir="$(realpath "$dest_dir")"
 # Copy the binaries to the site directory.
 cd "$(dirname "$output_path")"
 for f in ./coder-slim_*; do
+	# Remove ./ prefix
 	f="${f#./}"
-	dest="$dest_dir/${f//-slim_$version/}"
+	# Remove "-slim_$version"
+	f="${f//-slim_$version/}"
+	# Replace underscores with hyphens
+	f="${string//_/-}"
```

Comment:
should this be `f="${f//_/-}"`?  Where does `string` come from?

A pattern that I like to use to when I'm doing a series of modifications to some input is to assign them to new variables to make it completely clear what's happening.  In this case I would write:
```
for initial in ./coder-slim_*; do
	# Remove "-slim_$version"
	truncated="${initial//-slim_$version/}"
	# Replace underscores with hyphens
	hyphened="${truncated//_/-}"
	dest="$dest_dir/$hyphened"
```

Reply:
Change the variable names to `f`, `name`, `truncated`, `hyphenated`